zenodo/modules/deposit/templates/zenodo_deposit/delete.html

```diff
@@ -44,41 +48,33 @@ <h1>Delete record</h1>
         <p><h4>Are you ABSOLUTELY sure?</h4> This action <strong>CANNOT</strong> be undone! Deleting a record will
         permanently delete the associated files from disk.</p>
       </div>
+      {% if record_past_deletion_time %}
+        <div class="alert alert-warning" style="background-color:#df6d69" align="center"
```

Comment:
```suggestion
        <div class="alert alert-error" align="center"
```

To get the same effect, but with the built-in Bootstrap CSS classes